streams/src/main/java/org/apache/kafka/streams/state/internals/LRUCacheEntry.java

```diff
@@ -102,5 +65,22 @@ public long size() {
         return sizeBytes;
     }
 
+    @Override
+    public boolean equals(final Object o) {
```

Comment:
Why do we need `equals()` and `hashCode()` now?

Reply:
To make `bugWarnings` happy

Reply:
Why was bugWarnings unhappy? Is this class used in an equals comparison somewhere?

Reply:
I think bugwarning requires any objects that is used as key in a map to override these two functions.